src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -261,12 +261,11 @@ private EventDescriptor IPropChangedEventValue
                     _state[s_bitIPropChangedQueried] = true;
```

Comment:
What constant are you referring to? `s_bitIPropChangedQueried` is used to limit the calculation of `_realIPropChangedEvent`. Otherwise `_realIPropChangedEvent` will be calculated every time. Right?

Reply:
`_realIPropChangedEvent` doesn't exist on `!FEATURE_IDENTIFYPROPERTYCHANGED` builds so on those configurations, we don't need a flag bit to limit recalc. There's no calc to start with.

Reply:
Oh. Duh. Thanks.